manifests/repo.pp

```diff
@@ -1,148 +1,235 @@
-# == Define: yum::repo
+# @summary Manage individual yum repo files in /etc/yum.repos.d
```

Comment:
Needs to `include ::yum`

Reply:
`include ::yum` causes a duplicate resource declaration:
```
     Failure/Error: include ::yum

     Puppet::PreformattedError:
       Evaluation Error: Error while evaluating a Resource Statement, Evaluation Error: Error while evaluating a Resource Statement, Duplicate declaration: Exec[clean_yum_cache] is already declared in file :2; cannot redeclare at /root/puppet/puppet-module-yum/spec/fixtures/modules/yum/manifests/init.pp:737 at /root/puppet/puppet-module-yum/spec/fixtures/modules/yum/manifests/init.pp:737:3 at line 7 on node learning.puppetlabs.vm
```